packages/browsh.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Browsh < Package
+  description 'A fully-modern text-based browser, rendering to TTY and browsers.'
+  homepage 'https://www.brow.sh'
+  @_ver = '1.6.4'
+  version @_ver
+  license 'LGPL-2.1'
+  compatibility 'all'
+
+  depends_on 'firefox'
```

Comment:
We need another browser to use this?

Reply:
This program is literally a text-mode proxy to a firefox instance.

Reply:
Maybe that should be in the description?  It currently appears to be a browser.  Sounds more like a Firefox addon.

Reply:
One could consider it a TUI browser that uses firefox as a backend.

Reply:
I think we don't need to consider it (my purpose for adding this package is to provide a convenient way for user to install)